plugins/woocommerce/src/RestApi/Routes/V4/Orders/Schema/OrderSchema.php

```diff
@@ -0,0 +1,746 @@
+				'phone'      => $order->get_shipping_phone(),
+			),
+			'payment_method'       => $order->get_payment_method(),
+			'payment_method_title' => $order->get_payment_method_title(),
+			'transaction_id'       => $order->get_transaction_id(),
+			'date_paid'            => wc_rest_prepare_date_response( $order->get_date_paid(), false ),
+			'date_paid_gmt'        => wc_rest_prepare_date_response( $order->get_date_paid() ),
+			'date_completed'       => wc_rest_prepare_date_response( $order->get_date_completed(), false ),
+			'date_completed_gmt'   => wc_rest_prepare_date_response( $order->get_date_completed() ),
+			'cart_hash'            => $order->get_cart_hash(),
+			'payment_url'          => $order->get_checkout_payment_url(),
+			'is_editable'          => $order->is_editable(),
+			'needs_payment'        => $order->needs_payment(),
+			'needs_processing'     => $order->needs_processing(),
+		);
+
+		if ( in_array( 'line_items', $include_fields, true ) ) {
+			$line_items         = $order->get_items( 'line_item' );
+			$data['line_items'] = array();
+			foreach ( $line_items as $line_item ) {
+				$data['line_items'][] = $this->order_item_schema->get_item_response( $line_item, $request );
+			}
+		}
+
+		if ( in_array( 'shipping_lines', $include_fields, true ) ) {
+			$line_items             = $order->get_items( 'shipping' );
+			$data['shipping_lines'] = array();
+			foreach ( $line_items as $line_item ) {
+				$data['shipping_lines'][] = $this->order_shipping_schema->get_item_response( $line_item, $request );
+			}
+		}
+
+		if ( in_array( 'coupon_lines', $include_fields, true ) ) {
+			$line_items           = $order->get_items( 'coupon' );
+			$data['coupon_lines'] = array();
+			foreach ( $line_items as $line_item ) {
+				$data['coupon_lines'][] = $this->order_coupon_schema->get_item_response( $line_item, $request );
+			}
+		}
+
+		if ( in_array( 'fee_lines', $include_fields, true ) ) {
+			$line_items        = $order->get_items( 'fee' );
+			$data['fee_lines'] = array();
+			foreach ( $line_items as $line_item ) {
+				$data['fee_lines'][] = $this->order_fee_schema->get_item_response( $line_item, $request );
+			}
+		}
+
+		if ( in_array( 'tax_lines', $include_fields, true ) ) {
+			$line_items        = $order->get_items( 'tax' );
+			$data['tax_lines'] = array();
+			foreach ( $line_items as $line_item ) {
+				$data['tax_lines'][] = $this->order_tax_schema->get_item_response( $line_item, $request );
+			}
+		}
+
+		if ( in_array( 'meta_data', $include_fields, true ) ) {
+			$filtered_meta_data = $this->filter_internal_meta_keys( $this->get_meta_data_for_response( $order->get_meta_data(), $request ) );
+			$data['meta_data']  = array();
+			foreach ( $filtered_meta_data as $meta_item ) {
+				$data['meta_data'][] = array(
+					'id'    => $meta_item->id,
+					'key'   => $meta_item->key,
+					'value' => $meta_item->value,
+				);
+			}
+		}
+
+		// Add COGS data.
+		if ( $this->cogs_is_enabled() ) {
```

Comment:
Should we check `$include_fields` before getting it like we do with other properties above?

Reply:
https://github.com/woocommerce/woocommerce/pull/60980/commits/2e3cb11111d550a636ac3d38c1af373c2293b58a